lib/lhm.rb

```diff
@@ -96,7 +96,7 @@ def setup(adapter)
   def adapter
     @@adapter ||=
       begin
-        raise 'Please call Lhm.setup' unless defined?(ActiveRecord)
+        fail 'Please call Lhm.setup' unless defined?(ActiveRecord)
```

Comment:
I hate this `fail` instead of `raise` on rubocop, I think this should be one we ignore!